voluncheer/voluncheer/settings.py

```diff
@@ -139,8 +141,11 @@
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
-STATIC_ROOT = os.path.join(BASE_DIR, "static")
-STATIC_URL = '/static/'
+# STATIC_ROOT = os.path.join(BASE_DIR, "static")
```

Comment:
Is this unused? If so it should be removed not commented out.

Reply:
Thanks for noticing, my mistake, I commented that only for local tests